sdk/servicebus/azure-servicebus/azure/servicebus/management/_management_client.py

```diff
@@ -335,6 +336,13 @@ def create_queue(self, queue_name, **kwargs):
 
         :rtype: ~azure.servicebus.management.QueueProperties
         """
+        forward_to = _normalize_forward_to_address(
+            kwargs.pop("forward_to", None), self.fully_qualified_namespace
+        )
+        forward_dead_lettered_messages_to = _normalize_forward_to_address(
+            kwargs.pop("forward_dead_lettered_messages_to", None),
+            self.fully_qualified_namespace,
+        )
```

Comment:
could you help check whether `update_*` methods should also apply the change?

Reply:
looks like JS does it: https://github.com/Azure/azure-sdk-for-js/blob/c81faf961d6eedf409470407e0f3f6b4b1621ea2/sdk/servicebus/service-bus/test/public/atomManagement.spec.ts#L2211. 
Java checks before setting forward to in all CRUD operations: https://github.com/Azure/azure-sdk-for-java/blob/8be0607215eea08527382d3f665f02b80af2572a/sdk/servicebus/microsoft-azure-servicebus/src/main/java/com/microsoft/azure/servicebus/management/SubscriptionDescription.java#L289
.NET does the check when normalizing the description, which can happen after create/update: https://github.com/Azure/azure-sdk-for-net/blob/8b0b10d6e0380d6f23941b0e864c1cf6d0657f02/sdk/servicebus/Microsoft.Azure.ServiceBus/src/Management/SubscriptionDescriptionExtensions.cs#L13
Will be updating to match.